Sources/App/Features/Push/Controllers/PushController.swift

```diff
@@ -43,4 +44,25 @@ struct PushController: RouteCollection {
 
         return .noContent
     }
+
+    private func sendNotification(req: Request) async throws -> HTTPStatus {
+        struct SendNotificationRequest: Decodable {
+            let message: String
+            let securityCode: String
+        }
+
+        let notificationRequest = try req.content.decode(SendNotificationRequest.self)
+        let secretSecurityCode = Environment.get("PUSH_SECURITY_CODE")
+
+        guard notificationRequest.securityCode == secretSecurityCode else { throw Abort(.forbidden) }
```

Comment:
Already discussed on Slack but I'm personally hesitant to support adding more security mechanisms. Having said that, it's something we can come back to and tweak once we have a better authentication solution.

Reply:
That was my thinking